src/resources/elements/tokenSelect/tokenSelect.scss

```diff
@@ -9,12 +9,20 @@
     align-items: center;
     height: 23px;
 
-    img {
+    img, i {
       @include tokenIcon;
       position: relative;
+      text-align: center
+    }
+
+    img {
       top: 2px;
     }
 
+    i {
+      margin-top: 2px;
```

Comment:
why aren't you using top or bottom, like `img` does, since the position is relative?

Reply:
reverted in commit 3bb80d3 as it is not necessary anymore.